server/game/core/card/propertyMixins/UnitProperties.ts

```diff
@@ -101,9 +126,13 @@ export function WithUnitProperties<TBaseClass extends InPlayCardConstructor>(Bas
             let triggeredAbilities = super.getTriggeredAbilities();
 
             // add any temporarily registered attack abilities from keywords
-            if (this._attackKeywordAbilities !== null || this.isBlank()) {
+            if (this._attackKeywordAbilities !== null || this.isBlank()) { //TODO: Why is this isBlank check even here?
```

Comment:
Just to explain - the isBlank stuff was ported over from L5R as a way to shut off abilities when something blanks the card. That's not in place for us yet, I need to do a pass to figure out how it will work in the swu rules. But I left a bunch of the references in place since I think they're pretty close to correct and right now they'll never be activated.